_posts/2023-6-20-credential-suppression.markdown

```diff
@@ -0,0 +1,113 @@
+---
+layout: post
+title: Leaked Credentials!
+date: 2023-6-20
```

Comment:
Once we are ready to commit, will need to update this to the day

Reply:
agreed